articles/rules/guides/redirect.md

```diff
@@ -108,74 +108,61 @@ Suppose you would like to force users to change their passwords under specific c
 1. The user attempts to log in and needs to change their password.
 2. The user is redirected to an application-specific page with a JWT in the query string. This JWT ensures that only this user's password can be changed and **must be validated** by the application.
 3. The user changes their password in the application-specific page by having the application call the [Auth0 Management API](/api/v2#!/Users/patch_users_by_id)
-4. The application redirects back to `/continue`, with a JWT in the query string. This token must be issued for the same user that is attempting to log in, and must contain a `passwordChanged: true` claim.
+4. Once the user has successfully changed their password, the application extracts the `authorize_again` claim from the verified and decoded JWT, then proceeds to redirect the user to that URL allowing them to sign in with their new password.
 
 ```js
 function(user, context, callback) {
-  // Prerequisites:
-  // 1. Implement a `mustChangePassword` function
-  // 2. Set configuration variables for the following:
-  // * CLIENT_ID
-  // * CLIENT_SECRET
-  // * ISSUER
-  if (context.protocol !== "redirect-callback") {
-    if (mustChangePassword(user)) {
-      // User has initiated a login and is forced to change their password
-      // Send user's information in a JWT to avoid tampering
-      function createToken(clientId, clientSecret, issuer, user) {
-        var options = {
-          expiresInMinutes: 5,
-          audience: clientId,
-          issuer: issuer
-        };
-        return jwt.sign(user, clientSecret, options);
-      }
-      var token = createToken(
-        configuration.CLIENT_ID,
-        configuration.CLIENT_SECRET,
-        configuration.ISSUER, {
-          sub: user.user_id,
-          email: user.email
-        }
-      );
-      context.redirect = {
-        url: "https://example.com/change-pw?token=" + token
+  /*
+   * Prerequisites:
+   * 1. Implement a `mustChangePassword` function
+   * 2. Set configuration variables for the following:
+   *    - CLIENT_ID
+   *    - CLIENT_SECRET
+   *    - ISSUER
+   */
+
+  const url = require('url@0.10.3');
+  const req = context.request;
+
+  function mustChangePassword() {
+    // TODO: implement function
```

Comment:
Quick question - was this meant to be left here

Reply:
@jeffreylees Yessir, that's meant to be here for the developer to implement the function. If it's causing confusion I can take it out - let me know :)

Reply:
No, all good, just making sure you meant it as a note for them and not yourself, I write TODOs in the docs all the time but for me 🤣